x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/mapper/Mapper.java

```diff
@@ -230,7 +244,7 @@ private PhysicalPlan mapMerge(Merge merge) {
     }
 
     public static boolean isPipelineBreaker(LogicalPlan p) {
-        return p instanceof Aggregate || p instanceof TopN || p instanceof Limit || p instanceof OrderBy;
+        return p instanceof Aggregate || p instanceof TopN || p instanceof Limit || p instanceof OrderBy || p instanceof Rerank;
```

Comment:
Just for me to understand:
- why did we make `Rerank` a pipeline breaker? can we not execute the `RerankOperator` on the data nodes? we are applying a sort immediately after so it might need to be a pipeline breaker?
- if `Rerank` is a pipeline breaker, does that mean CCS should be supported? but just have not tested it yet?

Reply:
Being compatible with CCS without any requirement on the model being deployed on the remote clusters is the main reason why I think it need to be a pipeline breaker.
I might revisit this when I will add some tests for it.

Reply:
I removed and will reconsider introducing it again as part of the CCS implementation.